index.html

```diff
@@ -201,12 +218,14 @@ <h3>Skills</h3>
         <ul><li><i class="fa-solid fa-angle-down"></i></li></ul>
       </div>
     </div>
+
+    </div>
   </section>
 
 
 
   <section class="contact-card">
-    <div class=" fifth-template  contact later">
+    <div class=" fifth-template  contact ">
       <h2 class="contact-header">Contact me</h2>
     <p class="contact-p">
       Hello I’m a software developer! I can help you build , a product , feature or website. Look through some of  my work and experience! If you like what you see  and have a project you need coded, don’t hesitate to contact me.
```

Comment:
- Kindly Remove  the border radius for input and textarea  to make them stick  to the figma design :+1: 
- Kindly center the submit button as shown in the figma design :+1: 
## Figma   design 

![image](https://user-images.githubusercontent.com/95297251/230360090-c9c9eb28-37bd-468c-ad28-309fec42d326.png)

## Your design 

![image](https://user-images.githubusercontent.com/95297251/230360380-d1416e60-c2be-459f-a60f-e07a3c54824b.png)